### ISSUE TITLE
Calculate ang4487aer and ang4487csaer from the components

### ISSUE THREAD

**Issue by jgriesfeller** [metno/aerocom-data-correction-tools#1]
IDL-code:
```
       if strmatch(c_var,'ANG4487_*') then begin
         iis55=where(tm eq 'OD550_'+c_species,i_counts55)
         iis44=where(tm eq 'OD440_'+c_species,i_counts44)
         iis82=where(tm eq 'OD825_'+c_species,i_counts82)
         iis86=where(tm eq 'OD865_'+c_species,i_counts86)
         iis87=where(tm eq 'OD870_'+c_species,i_counts87)
         if ( i_counts55 gt 0 or i_counts44 gt 0 ) and ( i_counts86 gt 0 or i_counts87 gt 0 or i_counts82 gt 0 ) then begin 
           if i_datatype eq 0 then tt=create_struct(tt,c_Var, fltarr(i_2DMVardims))
           if i_datatype eq 1 then tt=create_struct(tt,c_Var, fltarr(i_2DDVardims))
           if i_datatype eq 2 then tt=create_struct(tt,c_Var, fltarr(i_2DHVardims))
           i_vindex=where( tag_names(tt) eq c_var)
           
           if	i_counts44 gt 0  then begin
             wavela=440. 
             iis=iis44		
           endif else if i_counts55 gt 0 then begin
             wavela=550. 
             iis=iis55
           endif else wavela=-1.

           if	i_counts86 gt 0  then begin
             wavelb=865. 
             iid=iis86		
           endif else if i_counts87 gt 0 then begin
             wavelb=870. 
             iid=iis87
           endif else if i_counts82 gt 0 then begin
             wavelb=825. 
             iid=iis82
           endif else wavelb=-1.

           ratio=wavela/wavelb
           if ratio gt 0 then $
              tt.(i_vindex)[*]= -1.0*alog(tt.(iis)[*]/tt.(iid)[*])/(alog(ratio))
         endif
       endif	
````

**Comment by jgriesfeller** [comment]
solved for standard ang4487aer using the routine [CalcAngstroem.py](https://github.com/metno/aerocom-data-correction-tools/blob/master/python3/CalcAngstroem/CalcAngstroem.py)

**Comment by jgriesfeller** [comment]
I could not find a single model in the aerocom data base that had clear sky optical depth at two wavelengths to calculate the clear sky angstroem exponent.
Quitting development for ang4487csaer therefore for now.

**Comment by jgriesfeller** [comment]
user model CAM53-Oslo_7310_MG15CLM45_5feb2017IHK_53OSLO_PD_UNTUNED for the CS angstroem calculation for testing

**Comment by jgriesfeller** [comment]
Solved for clear sky also with commit ecf12c145830873391297115a1dd950da1f9cc97
